config/api.yml

```diff
@@ -942,7 +942,7 @@
     :resource_actions:
       :get:
       - :name: read
-        :identifier: miq_request_show
+        :identifier: miq_request_view
```

Comment:
why this change ?  miq_request_view is for the request view tab, vs. the miq_request_show_list (for collection list above) or displaying an individual request (e.g. resource) via miq_request_show.   At least that's what I understand from the miq_product_features.yml file.